### ISSUE TITLE
puppet_languageserver.rb:218:in `init_puppet': uninitialized constant 

### ISSUE THREAD

**Issue by skrysmanski** [puppetlabs/puppet-vscode#775]
### What Versions are you running?

OS Version: Windows 10 20H2
VSCode Version: 1.60.2
Puppet Extension Version: 1.3.0
PDK Version: ?
Puppet: 7.11.0

### What You Are Seeing?

When starting Visual Studio Code, the puppet extension tries to start the language server but each attempt fails with:

```
D:/Dev/Git/VSCode/VSCode-Extensions/extensions/puppet.puppet-vscode-1.3.0/vendor/languageserver/lib/puppet_languageserver.rb:218:in `init_puppet': uninitialized constant PuppetLanguageServer::Facter (NameError)
	from d:/Users/t02836a/.vscode/extensions/puppet.puppet-vscode-1.3.0/vendor/languageserver/puppet-languageserver:13:in `<main>'
[Info  - 07:21:56] Connection to server got closed. Server will restart.
```

After 5 attempts, Visual Studio Code gives up.

### What is Expected?

No errors.

### How Did You Get This To Happen? (Steps to Reproduce)

It happens directly when I start Visual Studio Code (and I guess if a Puppet related file is open(ed)).

### Output Log

Let's see first whether you guys actually need this.

**Comment by da-ar** [comment]
Hi @skrysmanski this is linked to the issue raised here: https://github.com/puppetlabs/puppet-editor-services/issues/311

**Comment by skrysmanski** [comment]
@da-ar Yeah, seems to be the same issue. For my side, this issue can be closed then.

**Comment by da-ar** [comment]
@skrysmanski I'll keep this issue open until it's resolved on the language server side for other how may come across the problem.

**Comment by skrysmanski** [comment]
@da-ar This seems to be fixed in 1.4.0 (at least I'm not getting the error anymore).

**Comment by da-ar** [comment]
> @da-ar This seems to be fixed in 1.4.0 (at least I'm not getting the error anymore).

Yes, thanks for the reminder. It was fixed with https://github.com/puppetlabs/puppet-editor-services/issues/311
